src/Helpers/PageHelpers/PageHelper.cs

```diff
@@ -406,6 +411,23 @@ private void CheckEmailAction(List<IAction> actions)
             });
         }
 
+        private void CheckValidateAction(List<IAction> actions)
+        {
+            if (!actions.Any())
+                return;
+
+            actions.ForEach(action =>
+            {
+                var foundSlug = action.Properties.PageActionSlugs.FirstOrDefault(_ => _.Environment.ToLower().Equals(_environment.EnvironmentName.ToS3EnvPrefix().ToLower()));
+
+                if (foundSlug == null)
+                    throw new ApplicationException($"PageHelper:CheckValidateAction, Validate there is no PageActionSlug for {_environment.EnvironmentName}");
+
+                if (string.IsNullOrEmpty(foundSlug.URL))
+                    throw new ApplicationException("PageHelper:CheckValidateAction, Validate action type does not contain a url");                
```

Comment:
can u add a check within here to make sure action.Properties.EHttpActionType is not Unknown.